src/fontstash.h

```diff
@@ -189,12 +190,19 @@ int fons__tt_getGlyphIndex(FONSttFontImpl *font, int codepoint)
 }
 
 int fons__tt_buildGlyphBitmap(FONSttFontImpl *font, int glyph, float size, float scale,
-							  int *advance, int *lsb, int *x0, int *y0, int *x1, int *y1)
+							  int oversampleH, int oversampleV,
+							  int *advance, int *lsb, int *x0, int *y0, int *x1, int *y1,
+	                          float *subX, float *subY)
 {
 	FT_Error ftError;
 	FT_GlyphSlot ftGlyph;
 	FONS_NOTUSED(scale);
 
+	// @@ How should we handle FreeType?
```

Comment:
I just want to be sure FreeType doesn't already do something equivalent in its default sub-pixel renderer or as an option.